### ISSUE TITLE
[Accessibility] The Office Fabric DetailsList headers are buttons with aria-sort label which isn't supported by button

### ISSUE THREAD

**Issue by SureneruS** [OfficeDev/office-ui-fabric-react#970]
### Bug Report

- __Package version(s)__: 1.2.2
- __Browser and OS versions__: not relevant

#### Describe the issue:

As title. It is failing ARIA standard. Can be fixed by removing the aria-sort attr from the component

`<button class="ms-DetailsHeader-cell is-actionable is-icon-visible" aria-haspopup="false" aria-label="Name" aria-sort="ascending" data-automationid="ColumnsHeaderColumn" data-item-key="name" tabindex="0" style="width: 416px;"><i class="ms-Icon ms-Icon--SortUp"></i><!-- react-text: 56 -->Name<!-- /react-text --></button>`

**Comment by dzearing** [comment]
@SureneruS is this something you can help address?

**Comment by dzearing** [comment]
If you are already testing for any aria issues, it would be helpful to submit PRs.

**Comment by SureneruS** [comment]
@dzearing yes, I can work on the fix.

**Comment by dzearing** [comment]
Thank you!
